Find the common symbol.md

```diff
@@ -0,0 +1,47 @@
+# Summary
+In this experiment each subject is given a card on which there appears a set of five (out of six possible) symbols. Each subject's card is different from all the others in that the symbol lacking is a different symbol in each case. Thus, in any set of five cards there is only one symbol in common. The task is for every subject to find the common symbol.
+
+# References
+Main paper: Leavitt, H. J. (1951). Some effects of certain communication patterns on group performance. The Journal of Abnormal and Social Psychology, 46(1), 38–50. https://doi.org/10.1037/h0057189
+
+# Stimuli
+## The visual components
+A card with a set of five out of six symbols as follows are presented to the subjects:
+
+![Alt text](/images/Find_the_common_symbol.jpg)
+
+# Procedure
+## Steps
+
+1. One hundred male undergraduates of M.I.T.,8 drawn from various classes at the Institute, served as subjects for these experiments.
+2. These 100 are split up into 20 groups of five men each. These 20 groups are then further subdivided so that five groups could be tested on each of four experimental patterns.
+3. In case the color or geographical position of one's work-space might affect one's behavior, positions are shifted for each new group.
+4. Each group is given 15 consecutive trials on one pattern, a process which required one session of about fifty minutes. These subjects are not used again. The order in which the patterns are used is randomized.
```

Comment:
I would add the following about how things were set up and a picture of the table: 

The 5 participants were seated around a circular table so that each was separated from the next by a vertical partition from the center to six inches beyond the table's edge. 

partitions had slots permitting subjects to push written message cards to the men on either side of them.

To allow for communication to the other men in the group, a five-layered pentagonal box was built and placed at the center of the table. The box was placed so that the partitions just touched each of the five points of the pentagon. Each ofthe five resulting wedge-shaped work-spaces was then painted a different color. 

The 5s were supplied with blank message cards whose colors matched that of their work spaces. Any message sent from a booth had to be on a card of the booth's color. 

On the left wall of each partition,16 large symbol cards, representing 16 trials, were hung in loose-leaf fashion. The cards were placed in order with numbered backs to S. At the starting signal  could pull down the first card and goto work.

In addition, each work space was provided with a board on which were mounted six switches.Above each switch appeared one of the six symbols.When 5 got an answer to the problem, he was to throw the proper switch, which would turn on an appropriate light on a master board of 30 lights in the observer's room.